Cargo.toml

```diff
@@ -16,6 +16,7 @@ all-features = true
 [dev-dependencies]
 criterion = "0.2"
 rand = "0.7"
+smallvec = "0.6"
```

Comment:
`smallvec` seems unused.

Reply:
it's used in benchmarks, but I think those benchmarks are no longer needed.